makederiv

```diff
@@ -520,6 +525,17 @@ while [[ "${@}" != "" ]] ; do
                 _add_video_filter "crop=720:480:0:32"
             fi
             _add_video_filter "yadif"
+	      
+		  elif [[ "${OUTPUT_TYPE}" == "streamable" ]]; then
+            MIDDLEOPTIONS+=(-movflags faststart)
+            MIDDLEOPTIONS+=(-pix_fmt yuv420p)
+            MIDDLEOPTIONS+=(-c:v libx264)
+			MIDDLEOPTIONS+=(-c:a aac)
```

Comment:
can you move the audio stuff to the later audio section and neaten the mixed indentation methods (4 spaces preferred)